src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -84,6 +84,15 @@ int init_sim() {
                 printf("HexagonWrapper::SetTracing failed: %d\n", status);
                 return -1;
             }
+        } else {
+            const char *T = getenv("HL_HEXAGON_SIM_MEM_TRACE");
+            if (T && T[0] != 0) {
```

Comment:
it because the argument to HL._HEXAGON_SIM_MEM_TRACE is a filename, its consistent with the other variants  of HL_HEXAGON_SIM*TRACE